src/azure-cli/azure/cli/command_modules/vm/generated/commands.py

```diff
@@ -16,14 +16,40 @@
 def load_command_table(self, _):
 
     from ..generated._client_factory import cf_ssh_public_key
-
     vm_ssh_public_key = CliCommandType(
         operations_tmpl='azure.mgmt.compute.operations._ssh_public_keys_operations#SshPublicKeysOperations.{}',
-        client_factory=cf_ssh_public_key,
-    )
+        client_factory=cf_ssh_public_key)
     with self.command_group('sshkey', vm_ssh_public_key, client_factory=cf_ssh_public_key) as g:
         g.custom_command('list', 'sshkey_list')
         g.custom_show_command('show', 'sshkey_show')
         g.custom_command('create', 'sshkey_create')
         g.custom_command('update', 'sshkey_update')
         g.custom_command('delete', 'sshkey_delete', confirmation=True)
+
+    from ..generated._client_factory import cf_gallery
+    vm_gallery = CliCommandType(
+        operations_tmpl='azure.mgmt.compute.operations._galleries_operations#GalleriesOperations.{}',
+        client_factory=cf_gallery)
+    with self.command_group('sig', vm_gallery, client_factory=cf_gallery) as g:
+        g.custom_command('image-definition list-shared', 'sig_image_definition_list_shared')
+        g.custom_command('image-version list-shared', 'sig_image_version_list_shared')
+        g.custom_command('list-shared', 'sig_list_shared')
+        g.custom_command('show-shared', 'sig_show_shared')
```

Comment:
Do we need to consider adding `is_experimental=True` for those two commands?

Reply:
the same as above